PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/StockViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  StockViewController.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by Sam Roman on 9/4/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
+//
+
+import UIKit
+
+class StockViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    //MARK: - Variables and Outlets
+    
+    var stockData: [Stock]!
+    var sectionNames = [String]()
+//    var matrixData = [[Stock]]()
+    
+    //MARK: - Stock Methods
+    //This Seperates stockData by sectionName into a new array of [Stock]
+    func stockBySection(sectionNumber: Int) -> [Stock] {
+        return stockData.filter({$0.sectionName == sectionNames[sectionNumber]})
+    }
+    
```

Comment:
This is good, but maybe should be in the model? Still impressed by the implemantation